pkg/doc/did/doc.go

```diff
@@ -1395,6 +1400,16 @@ func populateRawServices(services []Service, didID, baseURI string) []map[string
 	return rawServices
 }
 
+func populateRawAlsoKnownAs(aka []string) []interface{} {
```

Comment:
please add unit tests to validate aka value as per schema

Reply:
suggest you cover a few different IRI formats, such as:

`did:example:123`, `https://social.example/username`, `urn:uuid:1231`

Reply:
See https://www.rfc-editor.org/rfc/rfc3986#section-1.1.3

Reply:
@Baha-sk Unit tests added to `doc_test.go`